adserver/views.py

```diff
@@ -818,6 +821,94 @@ def get_success_url(self):
         )
 
 
+class PublisherStripeOauthConnectView(
+    PublisherAccessMixin, UserPassesTestMixin, RedirectView
+):
+
+    """Redirect the user to the correct Stripe connect URL for the publisher."""
+
+    model = Publisher
+    permanent = False
+
+    def get_redirect_url(self, *args, **kwargs):
+        if not settings.STRIPE_CONNECT_CLIENT_ID:
+            messages.error(self.request, _("Stripe is not configured"))
+            return reverse("dashboard-home")
+
+        publisher = self.get_object()
+
+        stripe_state = get_random_string(30)
```

Comment:
Might be useful to explain this a bit. I assume it's a secret we verify at some point?

Reply:
That's fair. I'll add a note. Yes this is just returned after the stripe oauth connect so we only save values that were redirected from stripe.